sdrbase/maincore.cpp

```diff
@@ -358,7 +358,9 @@ void MainCore::initPosition()
     if (m_positionSource)
     {
         connect(m_positionSource, &QGeoPositionInfoSource::positionUpdated, this, &MainCore::positionUpdated);
+        #if (QT_VERSION < QT_VERSION_CHECK(6, 0, 0))
         connect(m_positionSource, &QGeoPositionInfoSource::updateTimeout, this, &MainCore::positionUpdateTimeout);
+        #endif
```

Comment:
no #else ?

Reply:
Since qt6 the signal updateTimeout is removed and got added to the errors enums as an erorr ,so no need for this connection,instead i made if condition in your error slot function  to call the slot function of timeout error